backend/corpora/common/corpora_orm.py

```diff
@@ -247,11 +228,11 @@ class DbDataset(Base):
     sex = Column(JSONB)
     ethnicity = Column(JSONB)
     development_stage = Column(JSONB)
-    source_data_location = Column(String)
     preprint_doi = Column(String)
     publication_doi = Column(String)
+    is_valid = Column(Boolean, default=False)
     project_id = Column(String, nullable=False)
```

Comment:
Should this be "collection_id" and "collection_visibility"

Reply:
fixed